package.json

```diff
@@ -359,6 +359,7 @@
     "jszip": "^3.2.2",
     "prettier-bytes": "^1.0.4",
     "qr-image": "^3.2.0",
+    "noble-bls12-381": "0.4.3",
```

Comment:
Have we done a security review of this library?

Reply:
posted https://github.com/brave/security/issues/691